oat-sa/expr-eval
# Fix/ADF-1568/Vulnerabilities in dependencies

### Related to: https://oat-sa.atlassian.net/browse/ADF-1568

### Summary

Quick fix for recent vulnerabilities discovered in dependencies.
Also, the lockfileVersion was updated to version 3, we now use Node18 for the min version (no need to hard base the requirement, the code is still compatible with older versions)

Additional changes:
- The coverage report has been fixed on the CI
- An auto-release action has been added

### How to test
- install the package `npm i` (please use Node18+)
- run the tests `npm test`
- check the vulnerabilities: `npm audit`
